packages/xdl/src/xdl.ts

```diff
@@ -20,8 +19,7 @@ export { Api };
 import ApiV2 from './ApiV2';
 export { ApiV2 };
 
-// @ts-ignore untyped module yet to be converted to TypeScript
-import * as AssetUtils from './AssetUtils.js';
+const AssetUtils: any = require('./AssetUtils.js');
```

Comment:
needs to be done this way because ts-ignore comments don't propagate to expo-cli and are interpreted as errors there

Reply:
I just merged #813, which adds types to AssetUtils so you should be able to remove this particular one.